### ISSUE TITLE
fetches only pre-releases of SYSLINUX

### ISSUE THREAD

**Issue by jonassmedegaard** [jsamr/bootiso#70]
If I read the code correctly, then when bootiso fetches SYSLINUX online it selects only between *prereleases* not final releases - i.e. it seemingly looks only below the base URL `https://mirrors.edge.kernel.org/pub/linux/utils/boot/syslinux/Testing` and the final releases are located one folder up from that.

**Comment by jsamr** [comment]
@jonassmedegaard good catch. Final releases are grouped by major directories in the URI path; perhaps it was an instance of developer laziness; can't remember! Anyway, that can certainly be enhanced.
